quic/s2n-quic-core/src/datagram/default.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// s2n-quic's default implementation of the datagram component
+
+use crate::{
+    datagram::unreliable_datagram::{ConnectionInfo, Endpoint, Packet, Receiver, Sender},
+    time::Timestamp,
+};
+use alloc::collections::VecDeque;
+use bytes::Bytes;
+use core::time::Duration;
+
+#[derive(Debug, Default)]
+pub struct Disabled;
+
+// Default capacity is zero right now since the feature is currently disabled.
+const DEFAULT_CAPACITY: usize = 0;
```

Comment:
The capacity should ideally be configurable on the provider struct.

Reply:
I was wondering about this too, but I'm not sure where this configuration would occur. Is there some other provider that takes in config options that I can look at?

Reply:
Yeah you can take a look at the default CID format: https://github.com/aws/s2n-quic/blob/3aa9e7cd827b82a070f41d322a21a2e07139c7b1/quic/s2n-quic/src/provider/connection_id.rs#L28